Documentation/Functions/Select.rst

```diff
@@ -72,7 +72,7 @@ pidInList
 
 ..  confval:: pidInList
     :name: select_pidInList
-    :type: *list of page\_ids* / :ref:`stdWrap`
+    :type: *list of page uids* / :ref:`stdWrap`
```

Comment:
Maybe "parent page uids" or "page pid's", because the sentence after it talks about pids instead of uids?

Reply:
pids are page uids